.travis.yml

```diff
@@ -33,6 +33,10 @@ cache:
 before_cache:
   - rm -f $HOME/.cache/pip/log/debug.log
 
+before_install:
+  - sudo apt-get update -qq
+  - sudo apt-get install -y unrar
+
```

Comment:
Should we disable the provider if unrar isn't installed? Or does it give good results even without unrar?